src/Form.tsx

```diff
@@ -356,6 +361,22 @@ const _Form: Form = React.forwardRef(
       submitTimeout.set(() => submit().catch(done));
     };
 
+    const handleReset = (e?: React.SyntheticEvent) => {
+      if (e && e.preventDefault && e.stopPropagation) {
+        e.preventDefault();
+        e.stopPropagation();
+      }
+      notify(onReset);
+      onChange?.(
+        {
+          ...schema?.getDefault(),
```

Comment:
I think we should not include the schema default. There is some nuance with how that's calculated that might require the user to have more control. Just `defaultValue` should be enough here, the user can pass in the schema.default as `defaultValue` if they want